jenkins-plugin/src/main/java/org/jenkinsci/plugins/pipeline/maven/dao/AbstractPipelineMavenPluginDao.java

```diff
@@ -662,9 +662,7 @@ protected synchronized void testDatabase() throws RuntimeSqlException {
                 "inner join MAVEN_DEPENDENCY on (MAVEN_DEPENDENCY.artifact_id = MAVEN_ARTIFACT.id and MAVEN_DEPENDENCY.ignore_upstream_triggers = false) \n" +
                 "inner join JENKINS_BUILD as downstream_build on MAVEN_DEPENDENCY.build_id = downstream_build.id \n" +
                 "inner join JENKINS_JOB as downstream_job on (downstream_build.number = downstream_job.last_successful_build_number and downstream_build.job_id = downstream_job.id) \n" +
-                "where MAVEN_ARTIFACT.group_id = ? and MAVEN_ARTIFACT.artifact_id = ? and MAVEN_ARTIFACT.version = ? \n" +
-                "and (MAVEN_ARTIFACT.type = ? or MAVEN_ARTIFACT.classifier IS NULL or MAVEN_ARTIFACT.classifier = ?) \n" +
-                "and downstream_job.jenkins_master_id = ?";
+                "where MAVEN_ARTIFACT.group_id = ?1 and MAVEN_ARTIFACT.artifact_id = ?2 and MAVEN_ARTIFACT.version = ?3 and MAVEN_ARTIFACT.type = ?4 and (MAVEN_ARTIFACT.classifier = ?5 or (MAVEN_ARTIFACT.classifier is null and ?5 is null)) and downstream_job.jenkins_master_id = ?6";
```

Comment:
sorry, this is totally subjective, but could you please split your where clause on different lines, as before. This would be much easier to read on Github HMI

Reply:
done :) (I hope my force-push doesn't mess up your review)